python/clean_export_aggregate.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import sys
+from sqlalchemy import create_engine
+
+BUILD_ENGINE = sys.argv[1]
+
+template_lookup = {
+    "aggregate_cdta_2020": "CDTA",
+    "aggregate_block_2020": "CensusBlocks",
+    "aggregate_tract_2020": "CensusTracts",
+    "aggregate_councildst_2010": "CityCouncil",
+    "aggregate_commntydst_2010": "CommunityDistricts",
+    "aggregate_nta_2020": "NTA",
+}
+
+
+def read_aggregate_template(name: str):
+    base = pd.read_csv(
+        f"data/agg_template/{template_lookup[name]}.csv", dtype=str)
+    base.set_index(get_index_columns(name),
+                   inplace=True, verify_integrity=True)
```

Comment:
Why do we verify integrity here? Seems like a fine choice to make I'm just curious why you made it

Reply:
Good questions. Because when I first tried running it. The `concat` failed because of nonunique index. As I was debugging added this and thought it might be okay to keep.